cinema/models.py

```diff
@@ -35,12 +39,19 @@ def full_name(self):
         return f"{self.first_name} {self.last_name}"
 
 
+def movie_image_file_path(instance, filename):
+    _, ext = os.path.splitext(filename)
+    filename = f"{slugify(instance.title)}-{uuid.uuid4()}.{ext}"
```

Comment:
Why you need dot between {uuid.uuid4()} and {ext}
Did you check is the ext has a dot or no?

Reply:
there is no dot  in Readme.md (f"{slugify(movie.title)}-{uuid}{ext}") but perhaps it is an error, as for me, since {ext} is a file extension, it should follow dot, like 123(.)jpg. Danylo's video also has dot.
<img width="485" alt="Screenshot 2023-05-12 at 20 12 52" src="https://github.com/mate-academy/py-images-and-custom-auth-serializer/assets/10571459/b500607d-7822-4d1d-939e-218772773510">